CRM/Civicase/Form/CaseWebforms.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/**
+ * This class generates form components for Civicase webforms
+ */
+class CRM_Civicase_Form_CaseWebforms extends CRM_Admin_Form {
+
+  private $webforms = array();
+
+  /**
+   * Builds the form object.
+   */
+  public function buildQuickForm() {
+    parent::buildQuickForm();
+    
+    $webforms = civicrm_api3('Case', 'getwebforms');
+    $errorMsg = null;
+    if (isset($webforms['values'])) {
+      foreach ($webforms['values'] as $item) {
+        $webformids[] = 'webforms_'.$item['nid'];
+        $this->add('checkbox', 'webforms_'.$item['nid'], $item['title']);
+        $this->webforms[$item['nid']] = $item;
+      }
+    }
+    else if(isset($webforms['is_error'])) {
+      $errorMsg = $webforms['error_message'];
+    }
+    else {
+      $errorMsg = ts('No Webforms with cases exists!');
+    }
+    $this->assign('nids', $webformids);
+    $this->assign('error', $errorMsg);
+
+    $this->addButtons([
```

Comment:
You're using here short-hand array syntax, while most of the rest of the code you added uses `array()` syntax to define arrays. Which one should it be?